### PR TITLE
chore: add enable k8s device plugin to cns config

### PR DESCRIPTION
**Pull request by jaer-tsun**
**Reason for Change**:
Adding EnableK8sDevicePlugin field in CNS config for future use.

Updated config in AKS-RP side https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/10476188

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by timraymond — DISMISSED** [review]

File: cns/configuration/configuration.go [hunk 1]
```diff
@@ -52,6 +52,7 @@ type CNSConfig struct {
 	WatchPods                   bool `json:"-"`
 	WireserverIP                string
 	GRPCSettings                GRPCSettings
+	EnableK8sDevicePlugin       bool
```
Comment (rbtr) [inline cns/configuration/configuration.go:55]: please, I have sorted this so many times

**Review by rbtr — CHANGES_REQUESTED** [review]

File: cns/configuration/configuration.go (hunk 1 above)
Comment (jaer-tsun) [reply cns/configuration/configuration.go:55]: ok moved it up

Comment (timraymond) [reply cns/configuration/configuration.go]: @jaer-tsun Bonus points: add a comment to the bottom of this struct saying something to the effect of "keep this list alphabetically sorted," so someone doesn't make the same mistake in the future.

**Review by rbtr — APPROVED** [review]

**Pull request merged**